templates/ss_env.erb

```diff
@@ -72,3 +72,9 @@ if(isset($_SERVER['SCRIPT_FILENAME'])) {
 if(file_exists('<%= @master_file_dir %>/_ss_environment_secrets.php')) {
 	require_once('<%= @master_file_dir %>/_ss_environment_secrets.php');
 }
+
+<%# Temporary fix for defining COMPOSER_HOME required to prevent a permission error when executing the UpdatePackageInfoTask in module bringyourownideas/silverstripe-maintenance %>
+<%# See issue https://github.com/bringyourownideas/silverstripe-maintenance/issues/121 %>
+if (!getenv('COMPOSER_HOME')) {
+	putenv('COMPOSER_HOME=/tmp');
```

Comment:
Please use `sys_get_temp_dir()` instead of hard-coded `/tmp`

Reply:
Thanks @stojg! 
I've updated ss_env.erb to use the sys_get_temp_dir() for defining the COMPOSER_HOME variable.